src/Templates/CartTemplate.php

```diff
@@ -13,8 +12,14 @@ class CartTemplate {
 
 	/**
 	 * Constructor.
+	 *
+	 * Templates require FSE theme support, so this will only init if a FSE theme is active.
```

Comment:
Let's use the term "Block theme" as it is the "official term" now. WP community is moving away from "FSE theme" (or [FSE anything for that matter](https://make.wordpress.org/updates/2022/11/04/site-editor-a-more-user-friendly-name/))